### DISCUSSION TITLE
Is it just me or verification for GitHub Student Pack is getting harder?

### DISCUSSION THREAD

**Discussion by k0mr4d3** [community/community#50467] [GitHub Education]
I'm applying again for GitHub Student Pack because my current subscription is expiring in 3 days. Tried adding my name to my GitHub bio, submitting my student ID card, my school portal, everything but still getting auto rejected. Last time I applied for this, I got accepted right away.

**Comment by sTiKyt** [comment]
It definitely is incredibly hard now, and they even removed the ability to contact a real person to check your application and bypass the automatic response, now they just ignore you.
